ivy/functional/ivy/linear_algebra.py

```diff
@@ -160,6 +160,22 @@ def cholesky(
     return current_backend(x).cholesky(x, upper=upper, out=out)
 
 
```

Comment:
Hello, you are adding frontend function you just can't edit or in ivy/functional you must add the function in frontend only as the cholesky_inverse is not common function in all frameworks you have to implement it's functionality in frontend

Reply:
Also you can check in the documentation the concept of adding frontend function (https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#ivy-frontends)